tools/class-parse/class-parse.csproj

```diff
@@ -12,11 +12,12 @@
     <ProductVersion>8.0.30703</ProductVersion>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <Import Project="..\..\Configuration.props" />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug</OutputPath>
+    <OutputPath>$(UtilityOutputPath)</OutputPath>
```

Comment:
True, but I believe that a value like `$(MSBuildThisFileDirectory)..\..\bin\$(Configuration)` would work, and would be a relative path. Or is `$(MSBuildThisFileDirectory)` an absolute path...?